src/test/java/com/capgemini/fs/apiCommunicator/coinGeckoCommunicator/CoinGeckoCommunicatorTest.java

```diff
@@ -0,0 +1,31 @@
+package com.capgemini.fs.apiCommunicator.coinGeckoCommunicator;
+
+import com.capgemini.fs.coindashboard.apiCommunicator.coinGeckoCommunicator.CoinGeckoCommunicator;
+import java.util.ArrayList;
+import org.junit.jupiter.api.Test;
+
+
+class CoinGeckoCommunicatorTest {
+
+  @Test
+  void getCurrentListing() {
+    var s = new CoinGeckoCommunicator();
+    ArrayList<String> coins = new ArrayList<>();
+    //coins.add("bitcoin");
+    ArrayList<String> vsCurrencies = new ArrayList<>();
+    vsCurrencies.add("usd");
+    //vsCurrencies.add("eur");
+    s.getCurrentListing(coins, vsCurrencies);
```

Comment:
There is no single assertion or Mockito verification in this test class. Without them, we actually don't check returned values and we don't verify flow of application by verifying what was called on mocks (if applicable).